themes/phil-coffe-theme/src/WEB-INF/src/resources-importer/journal/structures/carousel_structure.json

```diff
@@ -0,0 +1,32 @@
+{
+    "availableLanguageIds": [
+        "en_US"
+    ],
+    "defaultLanguageId": "en_US",
+    "fields": [
+        {
+            "label": {
+                "en_US": "Image"
+            },
+            "predefinedValue": {
+                "en_US": ""
+            },
+            "style": {
+                "en_US": ""
+            },
+            "tip": {
+                "en_US": ""
+            },
+            "dataType": "image",
+            "fieldNamespace": "ddm",
+            "indexType": "keyword",
+            "localizable": true,
+            "name": "Image1kd4",
```

Comment:
We should rename this structure field to something custom which can be understood by the developer rather than keeping the default name